### PR TITLE
fix issue88

### PR DESCRIPTION
**Pull request by gaoxinge**
**Which issue(s) this PR fixes**:

Fixes https://github.com/apache/dubbo-go/issues/88.

**Does this PR introduce a user-facing change?**:

```release-note
none
```

### CODE REVIEW

**Comment by AlexStocks** [comment]
LGTM

File: common/logger/logger_test.go [hunk 1]
```diff
@@ -41,7 +42,7 @@ func TestInitLog(t *testing.T) {
 	path, err = filepath.Abs("./logger.yml")
 	assert.NoError(t, err)
 	err = InitLog(path)
-	assert.EqualError(t, err, "ioutil.ReadFile(file:"+path+") = error:open "+path+": no such file or directory")
+	assert.Contains(t, err.Error(), fmt.Sprintf("ioutil.ReadFile(file:%s) = error:", path))
```
Comment (fangyincheng) [inline common/logger/logger_test.go]: I think message of error should be printed.  You can decide which to use based on the operating system.

Comment (gaoxinge) [reply common/logger/logger_test.go]: @fangyincheng Ok, I try to use os info to fix this.

File: common/logger/logger_test.go [hunk 2]
```diff
@@ -20,6 +20,7 @@ package logger
 import (
 	"fmt"
 	"path/filepath"
+	"runtime"
 	"testing"
```
Comment (AlexStocks) [inline common/logger/logger_test.go]: pls use this package in another import clause

Comment (gaoxinge) [reply common/logger/logger_test.go]: I have added a new import clause.

Comment (AlexStocks) [reply common/logger/logger_test.go]: LGTM

**Comment by AlexStocks** [comment]
LGTM

**Pull request merged**